tock/tock/static/js/components/timecard.js

```diff
@@ -235,7 +235,12 @@ function addEntry() {
   let fields = newEntry.querySelectorAll('input, select, textarea');
   // remove values
   for (f of fields) {
-    f.value = '';
+    if (f.matches('[type="checkbox"]')) {
```

Comment:
it might be nicer to just actively select the delete input and set it to false?

```
fields.querySelector('.entry-delete input').checked = false
```

Reply:
My thought was that in either case we needed excluded the checkbox input from `fields` when setting `value=''` and this was easier in the moment than excluding checkboxes from the querySelectorAll above 😅 

I'm not too attached to either implementation but this works as well --What do you think 👍  👎 ❓

```javascript
  let non_checkbox_fields = newEntry.querySelectorAll('input:not([type="checkbox"]), select, textarea');
  // remove values
  for (f of non_checkbox_fields) {
      f.value = '';
  }
  // Delete input must be unchecked
  newEntry.querySelector('.entry-delete input').checked = false
```

Reply:
Went with the above approach to handle checkboxes independently of the other fields we're cloning. Thanks @Sgtpluck!